spec.html

```diff
@@ -32162,26 +32223,92 @@ <h1>
         </emu-note>
       </emu-clause>
 
-      <emu-clause id="sec-defaulttimezone" type="implementation-defined abstract operation">
-        <h1>DefaultTimeZone ( ): a String</h1>
+      <emu-clause id="sec-availabletimezoneidentifiers" type="implementation-defined abstract operation">
+        <h1>AvailableTimeZoneIdentifiers ( ): a List of Records with fields [[Identifier]] (a String value that is a time zone identifier) and [[PrimaryIdentifier]] (a String value that is a primary time zone identifier)</h1>
```

Comment:
```suggestion
        <h1>AvailableTimeZoneIdentifiers ( ): a List of Records with fields [[Identifier]] (a String that is a time zone identifier) and [[PrimaryIdentifier]] (a String that is a primary time zone identifier)</h1>
```

Reply:
Also, it might be worth declaring a name for this kind of Record (with a 'Fields' table), as writing it out in full is a bit verbose.

Reply:
Good idea. New table is now added as `#sec-time-zone-identifier-record`. @jmdyck want to take a look?

Reply:
Yeah, that looks like an improvement to me.